### ISSUE TITLE
rollem+rollup-plugin-multi-entry cause error

### ISSUE THREAD

**Issue by ShiMeiWo** [bahmutov/rollem#26]
I made rollem.config.js:
```
import typescript from 'rollup-plugin-typescript';
import multiEntry from 'rollup-plugin-multi-entry';

export default [{
  entry: './src/scripts/Main.ts',
  dest: './dist/index.js',
  format: 'iife',
  plugins: [typescript()]
}, {
  entry: 'test/**/*.ts',
  dest: './dist/test.js',
  format: 'es',
  plugins: [
    multiEntry(),
    typescript()
  ]
}];
```

and executed this command:
`rollem -c rollem.config.js --watch`

then I received:
```
Treating 'rollup-plugin-typescript' as external dependency
Treating 'rollup-plugin-node-resolve' as external dependency
Treating 'rollup-plugin-commonjs' as external dependency
Treating 'rollup-plugin-json' as external dependency
Treating 'rollup-plugin-multi-entry' as external dependency
[06:30:31 GMT+0000 (UTC)] watching source folders for changes [ 'src/scripts', 'test/**' ]
/home/ubuntu/workspace/node_modules/watch/main.js:73
    if (err) throw err;
             ^

Error: ENOENT: no such file or directory, stat 'test/**'
```

Please check this issue.
Thanks.

**Comment by meszaros-lajos-gyorgy** [comment]
Rollem should have it's watcher behavior changed to expecting entries to be multi entries. The sources should be piped through node's glob module. I'll update my pull request for issue #25 so that it provides a solution to this too.

**Comment by meszaros-lajos-gyorgy** [comment]
This issue should be fixed with the latest 1.10 release. Please retest this issue.

**Comment by ShiMeiWo** [comment]
@meszaros-lajos-gyorgy
I've tested this issue just now and success to compile.
Thank you for your help!
